### DISCUSSION TITLE
[委托]学业困难

### DISCUSSION THREAD

**Discussion by FallingLine** [share121/inter-knot#3246] [其他]
求助！！这两道数学物理方法题该怎么写？！
![Uploading IMG_20240820_083545.jpg…]()
![Uploading IMG_20240820_083558.jpg…]()

**Comment by ColdLinK7** [comment]
看不到喵

**Comment by yzysrewrwqr** [comment]
<img src="https://upload-bbs.miyoushe.com/upload/2023/03/09/3173bc3e69fcfebecfda741340c9ad22_6841250310756275678.png" alt="开拓者 吃瓜ing" width="75" >自己写

**Comment by isle200inf** [comment]
你是不是图片还没上传完就发出来了<img src="https://upload-bbs.miyoushe.com/upload/2023/03/09/3173bc3e69fcfebecfda741340c9ad22_6841250310756275678.png" alt="开拓者 吃瓜ing" width="75" >
